### PR TITLE
CORE-16623 : Sync interop syntax with release

### PR DESCRIPTION
**Pull request by SaurabhR3**

### CODE REVIEW

**Review by jmacmahonr3 — APPROVED** [review]
LGTM

File: gradle.properties [hunk 1]
```diff
@@ -11,7 +11,7 @@ cordaProductVersion = 5.1.0-INTEROP
 ##   a per module property in which case module versions can change independently.
 ## IMPORTANT:
 ## The interop feature branches track api revisions separately to the mainline branch.
-## API version of last merge from corda mainline: 12
+## API version of last merge from corda mainline: 16
 cordaApiRevision = 15
```
Comment (szymonsztuka) [inline gradle.properties]: We should increment our version to 16 (`cordaApiRevision = 15` -> `cordaApiRevision = 16`)

Comment (ac101m) [reply gradle.properties]: Yeah, this is the only issue I can see

**Review by ac101m — APPROVED** [review]
LGTM

**Review by szymonsztuka — APPROVED** [review]

**Pull request merged**